assets/Zed/js/modules/main.js

```diff
@@ -6,7 +6,7 @@
 'use strict';
```

Comment:
Please get an approve from our FE-developer for all FE-related things (even if it's just renaming).